api/types/database.go

```diff
@@ -523,6 +524,7 @@ func (d *DatabaseV3) CheckAndSetDefaults() error {
 	case awsutils.IsRDSEndpoint(d.Spec.URI):
 		details, err := awsutils.ParseRDSEndpoint(d.Spec.URI)
 		if err != nil {
+			err = trace.Wrap(err)
```

Comment:
Why the error is wrapped with `trace.Wrap(err)` directly before `logrus.WithError(err)` call ?

Reply:
I thought we should wrap before logging the error, but I guess it doesn't matter since that func will have already wrapped all errors.

Reply:
removed wrapping